### PR TITLE
Update vis app names and routes

### PR DESCRIPTION
**Pull request by JackWilb**

### CODE REVIEW

**Comment by waxlamp** [comment]
@JackWilb, I marked this as Draft until we have a comprehensive name change strategy.

File: src/views/FrontPage.vue [hunk 1]
```diff
@@ -152,43 +152,43 @@ export default Vue.extend({
...
     samples(): Array<{title: string; image: NodeRequire; text: string; href: string }> {
       return [
         {
-          title: 'Paul Revere - Node Link Diagram',
+          title: 'Paul Revere - MultiLink',
           // eslint-disable-next-line global-require
           image: require('../assets/placard/boston.jpg'),
-          text: 'Explore the Paul Revere dataset using an interactive and beautiful node-link diagram. Discover the figures coordinating a pivotal event in history!',
-          href: `${this.nodelinkLink}/?workspace=boston&graph=boston`,
+          text: 'Explore the Paul Revere dataset using an interactive and beautiful MultiLink diagram. Discover the figures coordinating a pivotal event in history!',
```
Comment (waxlamp) [inline src/views/FrontPage.vue:169]: This instance should remain as it was (`MultiLink diagram` doesn't really mean anything, and in this description it's ok to use established vis vocabulary to say what the user will be seeing).

```suggestion
          text: 'Explore the Paul Revere dataset using an interactive and beautiful node-link diagram. Discover the figures coordinating a pivotal event in history!',
```

File: src/views/FrontPage.vue [hunk 2]
```diff
@@ -152,43 +152,43 @@ export default Vue.extend({
...
+          href: `${this.multiLinkURL}/?workspace=boston&graph=boston`,
         },
         {
-          title: 'Les Miserables - Adjacency Matrix',
+          title: 'Les Miserables - MultiMatrix',
           // eslint-disable-next-line global-require
           image: require('../assets/placard/miserables.jpg'),
-          text: 'Explore the Les Miserables dataset using an interactive adjacency matrix. See the factions and relationships for yourself!',
-          href: `${this.adjmatrixLink}/?workspace=miserables&graph=miserables`,
+          text: 'Explore the Les Miserables dataset using an interactive MultiMatrix. See the factions and relationships for yourself!',
```
Comment (waxlamp) [inline src/views/FrontPage.vue:176]: Same idea here:

```suggestion
          text: 'Explore the Les Miserables dataset using an interactive adjacency matrix. See the factions and relationships for yourself!',
```

File: src/views/FrontPage.vue [hunk 3]
```diff
@@ -152,43 +152,43 @@ export default Vue.extend({
...
+          href: `${this.multiMatrixURL}/?workspace=miserables&graph=miserables`,
         },
         {
-          title: 'Les Miserables - Node Link Diagram',
+          title: 'Les Miserables - MultiLink',
           // eslint-disable-next-line global-require
           image: require('../assets/placard/miserables2.jpg'),
-          text: 'The characters of Les Miserables, laid out in a colorful and interactive node link diagram.',
-          href: `${this.nodelinkLink}/?workspace=miserables&graph=miserables`,
+          text: 'The characters of Les Miserables, laid out in a colorful and interactive MultiLink diagram.',
```
Comment (waxlamp) [inline src/views/FrontPage.vue:183]:
```suggestion
          text: 'The characters of Les Miserables, laid out in a colorful and interactive node-link diagram.',
```

File: src/views/FrontPage.vue [hunk 4]
```diff
@@ -152,43 +152,43 @@ export default Vue.extend({
...
+          href: `${this.multiLinkURL}/?workspace=miserables&graph=miserables`,
         },
         {
-          title: 'Paul Revere - Adjacency Matrix',
+          title: 'Paul Revere - MultiMatrix',
           // eslint-disable-next-line global-require
           image: require('../assets/placard/boston2.jpg'),
-          text: 'See the relationships between Paul Revere and his contemporaries through an adjacency matrix layout.',
-          href: `${this.adjmatrixLink}/?workspace=boston&graph=boston`,
+          text: 'See the relationships between Paul Revere and his contemporaries through a MultiMatrix layout.',
```
Comment (waxlamp) [inline src/views/FrontPage.vue:190]:
```suggestion
          text: 'See the relationships between Paul Revere and his contemporaries through an adjacency matrix layout.',
```

**Review by waxlamp — CHANGES_REQUESTED** [review]
Looks good - a few suggestions for clarity to an end user.

File: src/views/FrontPage.vue (hunk 1 above)
Comment (JackWilb) [reply src/views/FrontPage.vue:169]: I thought the whole point of the MultiLink app was that it's not just a node-link diagram. I was trying to be more precise with the wording and not set people up to expect a standard node-link diagram

Comment (waxlamp) [reply src/views/FrontPage.vue:169]: That is true but calling it a "MultiLink diagram" doesn't really convey any information. It is in fact a node-link diagram, plus some extras. For an intro text blurb that is meant to get you to click on something, "node-link diagram" is the correct level of description for MultiLink.

**Review by waxlamp — APPROVED** [review]

**Pull request merged**